### ISSUE TITLE
Add a warning about old symfony installer version

### ISSUE THREAD

**Issue by jzawadzki** [symfony/symfony-installer#220]
Hi,

I noticed that on current symfony upgrade many users are rasing issues here that symfony installer doesn't work correctly. Most of those issues were already fixed but users haven't upgraded installer itself.

Do you think we should add a warning like composer have?
```
 Warning: This development build of composer is over 60 days old. It is recommended to update it by running "composer self-update" to get the latest version
```
 
And if so - how we should approach it? 
Composer compiles phar itself and adds constant with timestamp. We are just updating whole file (but only if there is newest version). 
Maybe we can just compare last modified time of phar with current time (+ `touch` file on `selfupdate` when its still up-to-date?)

**Comment by Pierstoval** [comment]
There is already a discussion about this in #136 , and even if the issue is closed, actually one of @xabbuh's comment is still pending for response :)

**Comment by ogizanagi** [comment]
I particulalry agree with @xabbuh's comment (https://github.com/symfony/symfony-installer/issues/136#issuecomment-89250313). I was not aware of this thread, and though about a similar idea with an error handler suggesting the user to update its installer if the version mismatches the latest one.

IMO, we've seen enough in the last days the need for such a feature. The installer should be pretty stable most of the time, but we cannot perfectly imagine how it would be in the future, or prevent all issues.

**Comment by Pierstoval** [comment]
The best solution IMO is to ask SensioLabs to add a simple "installer_version" field in versions.json that would be updated on each release with a simple hook. It's possible, but it needs some setup :)

**Comment by javiereguiluz** [comment]
The installer version is published in `get.symfony.com/symfony.version`, which is used by the Installer itself.

**Comment by Pierstoval** [comment]
Ok, then it should be easy to do!

**Comment by jzawadzki** [comment]
first draft up - please comment - and Happy SymfonyCon Hackday everyone :)

**Comment by chepeftw** [comment]
Hello! I was working on an update for this issue too, I got problems with my wifi on Saturday so I was not able to commit, I was thinking in just adding a function to DownloadCommand class, and the add the call to that function in the chain of methods in the execute method for DemoCommand and NewCommand. Do you think is worth a 3rd PR for the same issue but different implementation?

**Comment by Pierstoval** [comment]
I think we should first wait for @javiereguiluz to merge #221 before :)

**Comment by xabbuh** [comment]
closing as this was fixed by #221
